lib/SPIRV/SPIRVReader.cpp

```diff
@@ -1085,6 +1085,37 @@ Value *SPIRVToLLVM::transConvertInst(SPIRVValue *BV, Function *F,
   case OpFConvert:
     CO = IsExt ? Instruction::FPExt : Instruction::FPTrunc;
     break;
+  case OpBitcast:
+    CO = Instruction::BitCast;
```

Comment:
Perhaps it would be nice to add a comment here to highlight the difference between `OpBitcast` and llvm's `bitcast` (repeating the PR description would work for me).

Reply:
Sure, done.